presto-cassandra/src/test/java/io/prestosql/plugin/cassandra/CassandraTestingUtils.java

```diff
@@ -197,23 +197,23 @@ public static void createTableAllTypesPartitionKey(CassandraSession session, Sch
                 "   typeinteger, " +
                 "   typelong, " +
                 // TODO: NOT YET SUPPORTED AS A PARTITION KEY
```

Comment:
Are these TODO still relevant?

Reply:
Yes, these TODO are still relevant. Correctly, it describes partition pruning.